applications/RANSApplication/custom_constitutive/rans_newtonian_2d_law.cpp

```diff
@@ -0,0 +1,119 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+// System includes
+#include <iostream>
+
+// External includes
+
+// Project includes
+#include "includes/cfd_variables.h"
+#include "includes/checks.h"
+
+// Application includes
+#include "custom_utilities/fluid_calculation_utilities.h"
+
+// Include base h
+#include "custom_constitutive/rans_newtonian_2d_law.h"
+
+namespace Kratos
+{
+//******************************CONSTRUCTOR*******************************************
+//************************************************************************************
+
+RansNewtonian2DLaw::RansNewtonian2DLaw() : BaseType()
+{
+}
+
+//******************************COPY CONSTRUCTOR**************************************
+//************************************************************************************
+
+RansNewtonian2DLaw::RansNewtonian2DLaw(const RansNewtonian2DLaw& rOther)
+    : BaseType(rOther)
+{
+}
+
+//********************************CLONE***********************************************
+//************************************************************************************
+
+ConstitutiveLaw::Pointer RansNewtonian2DLaw::Clone() const
+{
+    return Kratos::make_shared<RansNewtonian2DLaw>(*this);
+}
+
+//*******************************DESTRUCTOR*******************************************
+//************************************************************************************
+
+RansNewtonian2DLaw::~RansNewtonian2DLaw()
+{
+}
+
+int RansNewtonian2DLaw::Check(
+    const Properties& rMaterialProperties,
+    const GeometryType& rElementGeometry,
+    const ProcessInfo& rCurrentProcessInfo)
+{
+    KRATOS_TRY
+
+    // Check viscosity value
+    KRATOS_ERROR_IF(rMaterialProperties[DYNAMIC_VISCOSITY] <= 0.0)
+        << "Incorrect or missing DYNAMIC_VISCOSITY provided in process info "
+           "for RansNewtonian2DLaw: "
+        << rMaterialProperties[DYNAMIC_VISCOSITY] << std::endl;
+
+    KRATOS_ERROR_IF(rMaterialProperties[DENSITY] <= 0.0)
+        << "Incorrect or missing DENSITY provided in process info "
```

Comment:
```suggestion
        << "Incorrect or missing DENSITY provided in material properties "
```
(I've just realised that this is also wrong in the Newtonian law)